### PR TITLE
[Docbug][resource][1718327]

### PR DESCRIPTION
**Pull request by Mahesh2516**
Removed **Handling incompatible clients** section.

### CODE REVIEW

**Review by Jagadeesh-MSFT — DISMISSED** [review]

**Review by Jagadeesh-MSFT — APPROVED** [review]

**Review by Wajeed-msft — DISMISSED** [review]

File: msteams-platform/resources/samesite-cookie-update.md [hunk 1]
```diff
@@ -36,41 +36,26 @@ Chrome 80, scheduled for release in February 2020, introduces new cookie values
...
 
 * Teams tabs use `<iframes>` to embed content that is viewed in a top-level or first-party context.
 * Task modules allow you to create modal popup experiences in your Teams application. Similar to a tab, a modal window opens inside the current page.
 * Message extensions allow you to insert enriched content into chat message from external resources.
 
-Any cookies used by embedded content will be considered third-party when the site is displayed in an `<iframe>`. In addition, if any remote resources on a page rely on cookies being sent with a request (e.g., `<img>` and `<script>` tags, external fonts, and personalized content) you'll need to ensure those are marked for cross-site usage — `SameSite=None; Secure` — or ensure that a fallback is in place.
+Any cookies used by embedded content must be considered third-party when the site is displayed in an `<iframe>`. In addition, if any remote resources on a page rely on cookies being sent with a request `<img>` and `<script>` tags, external fonts, and personalized content, you must ensure those are marked for cross-site usage, such as `SameSite=None; Secure` or ensure that a fallback is in place.
```
Comment (aosolis) [inline msteams-platform/resources/samesite-cookie-update.md:58]: "will" was correct here

**Review by surbhigupta12 — APPROVED** [review]
Approved the PR

**Review by KirtiPereira — APPROVED** [review]
Approved

**Comment by KirtiPereira** [comment]
@aosolis - please approve the PR.
Thanks.

**Review by Wajeed-msft — APPROVED** [review]

**Review by aosolis — APPROVED** [review]

**Pull request merged**